src/components/ProtectedDataCard.tsx

```diff
@@ -1,26 +1,35 @@
-import { useNavigate } from 'react-router-dom'
-import './ProtectedDataCard.css'
-import { Box, Card, CardContent, Chip, Divider } from '@mui/material'
+import { useNavigate } from 'react-router-dom';
+import './ProtectedDataCard.css';
+import { Box, Card, CardContent, Chip, Divider } from '@mui/material';
+import { DataSchema } from '@iexec/dataprotector';
+import { hasKey } from '../utils/utils';
 
 export interface ProtectedDataProps {
-  title: string
-  date: string
-  dataType: string
-  id: string
+  id: string;
+  title: string;
+  schema: DataSchema;
 }
 
 export default function ProtectedDataCard(props: ProtectedDataProps) {
-  const naviguate = useNavigate()
+  const navigate = useNavigate();
   return (
     <Card
       sx={{ minWidth: 260, cursor: 'pointer' }}
-      onClick={() => naviguate(`/protectedData/consent/${props.id}`)}
+      onClick={() => navigate(`/protectedData/consent/${props.id}`)}
     >
       <CardContent id="cardContent">
         <Box
           sx={{ height: '60px', display: 'flex', justifyContent: 'flex-end' }}
         >
-          <Chip id="chipLabel" label={props.dataType} size="small" />
+          <Chip
+            id="chipLabel"
+            label={
+              (hasKey(props.schema, 'email') && 'Email') ||
```

Comment:
i didn't understand why you used both haskey() and 'Email ?

Reply:
I make that to know if the keys email or file is present in the schema of the protected Data to display it if it's the case